packages/squarev2-integration/src/SquareV2PaymentMethod.tsx

```diff
@@ -178,12 +179,14 @@ const SquareV2PaymentMethod: FunctionComponent<PaymentMethodProps> = ({
     }, [deinitializePayment, initializePayment]);
 
     return (
-        <div className="loadingSpinner">
-            <LoadingOverlay isLoading={checkoutState.statuses.isInitializingPayment(method.id)}>
-                {renderPlaceholderFields()}
-                <div id={containerId} style={{ minHeight: '100px' }} />
-            </LoadingOverlay>
-        </div>
+        <SquareV2Form
+            checkoutState={checkoutState}
+            containerId={containerId}
+            deinitializePayment={deinitializePayment}
+            initializePayment={initializePayment}
+            method={method}
+            renderPlaceholderFields={renderPlaceholderFields}
```

Comment:
You can move `renderPlaceholderFields` inside `SquareV2Form`, because it is not related to any props from parent component

Reply:
fixed